### DISCUSSION TITLE
Custom Special Queries

### DISCUSSION THREAD

**Discussion by arda618** [searxng/searxng#2429] [Q&A] [answered]
I have asked GPT 3.5 about Custom Special Queries for a workaround to make the site: operator a hidden search parameter in every search (the enabled engines support it natively), but the solution it has provided is either outdated or wrong.

I need to know
- How to register a custom special query by modifying the source code (as seen in preferences)
- How to make that custom special query enabled by default

Please don't move this to Discussions, no one is answering questions there...

**Comment by arda618** [comment]
The example usage of the site: operator is
(user input) [hidden input in each search query: site:youtube.com OR site:dailymotion.com OR site:vimeo.com OR site:rumble.com]

**Comment by return42** [comment]
> Please don't move this to Discussions, no one is answering questions there...

Sorry, this is not acceptable, this is the issue tracker --> I move this thread into the discussions.

**Comment by arda618** [comment, marked as answer]
No hope. Thank you
